low-level/framework/utils/consulstore.py

```diff
@@ -74,20 +75,22 @@ def put(self, value, key, pickled=True):
                     .format(gerr, key))
                 break
 
-    def get(self, key, **kwargs):
-        """ Load data from given key"""
+    def _consul_get(self, key, **kwargs):
+        """Load consul data from the given key."""
         data = None
+        status = "Failure"
+
         for retry_index in range(0, MAX_CONSUL_RETRY):
             try:
-                _opt_recurse = kwargs.get("recurse", False)
                 key = self._get_key(key)
-                data = self.consul_conn.kv.get(key, recurse=_opt_recurse)[1]
```

Comment:
why removing the second argument here ? Its currently serving get() code which has very wide impact, have you thought of all scenarios ?

Reply:
By mistake, I removed second argument in last commit (copy-paste mistake). Thanks for pointing out.

Reply:
Corrected